src/page-modules/assistant/layout.tsx

```diff
@@ -54,6 +59,15 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
     getTransportModeFilter,
   );
 
+  useEffect(() => {
+    if (tripQuery.transportModeFilter === null)
+      onTransportFilterChanged(
+        transportModeFilter
```

Comment:
Could this logic be added directly to the `onTransportFilterChanged` and this hook can be removed?

Reply:
I am not sure that is possible, as the transportModeFilter initially is null before the useSWRImmutable() have returned the results.

Reply:
There are also other reasons that I did not see before.

But this should be fixed:
`ESLint: React Hook useEffect has missing dependencies: 'onTransportFilterChanged' and 'tripQuery.transportModeFilter'. Either include them or remove the dependency array.(react-hooks/exhaustive-deps)`